docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -359,6 +362,80 @@ rules:
 
 To deploy Beats in clusters with the Pod Security Policy admission controller enabled, or in OpenShift clusters, you must grant additional permissions to the Service Account used by the Beat Pods. Those Service Accounts must be bound to a Role or ClusterRole that has `use` permission for the required Pod Security Policy or Security Context Constraints. Different Beats and their features might require different settings set in their PSP/SCC.
 
+
+[id="{p}-beat-configuration-examples"]
+== Configuration Examples
+
+This directory contains yaml manifests with example configurations for Beats. These manifests are self-contained and work out-of-the-box on any non-secured Kubernetes cluster. All of them contain three-node Elasticsearch cluster and single Kibana instance. All Beat configurations set up Kibana dashboards if they are available for a given Beat and all required RBAC resources.
+
+==== Metricbeat for Kubernetes monitoring
+
+```yaml
+kubectl apply -f https://raw.githubusercontent.com/elastic/cloud-on-k8s/1.2/config/recipes/beats/metricbeat_hosts.yaml
+```
```

Comment:
I would recommend doing something like the following where you define a `beats_url` attribute and use that instead of repeating the URL in every sample.

```suggestion
:beats_url: https://raw.githubusercontent.com/elastic/cloud-on-k8s/{eck_release_branch}/config/recipes/beats

[source,sh,subs="attributes"]
----
kubectl apply -f {beats_url}/metricbeat_hosts.yaml
----

```